.github/workflows/delete-auth0-inactve-users.yml

```diff
@@ -0,0 +1,21 @@
+name: Delete dormant Auth0 users
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * MON
+
+jobs:
+  delete-dormant-auth0-users:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+      - run: python3 -m pip install -r python/requirements.txt
+      - run: python3 -m python.scripts.auth0_delete_inactive_users
+        env:
+          LOGGING_LEVEL: ${{ secrets.LOGGING_LEVEL }}
+          AUTH0_CLIENT_ID: ${{ secrets.AUTH0_CLIENT_ID }}
+          AUTH0_CLIENT_SECRET: ${{ secrets.AUTH0_CLIENT_SECRET }}
+          AUTH0_DOMAIN: ${{ secrets.AUTH0_DOMAIN}}
```

Comment:
Can you add a new line to the end of this file? Everything else looks great.

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline